Gems/LuaScript/Code/Source/Previewer/LuaScriptPreviewerFactory.cpp

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) Huawei Technologies Co., Ltd. 2021-2021. All rights reserved.
+ * Description: LuaScriptPreviewerFactory.
+ * Create: 2021-07-14
+ */
+
```

Comment:
Please see: 
https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#copyright-header-requirements
for copyright header requirements